packages/core/services/HttpServiceBase/index.ts

```diff
@@ -167,6 +167,19 @@ export default class HttpServiceBase {
         return new RestServiceResponse(cachedResponseData);
     }
 
+    public getEnvironmentFromUrl(): Environment {
```

Comment:
has unit test cov?

Reply:
removed in [3bee712](https://github.com/AllenInstitute/biofile-finder/pull/389/commits/3bee712c637ad2bf7571bed19b960fe11de85598)